LabExT/View/Movement/MoveStagesDeviceWindow.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+
+from tkinter import Frame, Label, Toplevel, Button, messagebox, RIGHT, TOP, X, BOTH, FLAT, Y
+from typing import Type
+from LabExT.Movement.Transformations import ChipCoordinate
+from LabExT.Movement.config import Axis
+from LabExT.Utils import run_with_wait_window
+
+from LabExT.Movement.MoverNew import MoverNew
+from LabExT.View.Controls.DeviceTable import DeviceTable
+from LabExT.View.Controls.ParameterTable import ConfigParameter, ParameterTable
+from LabExT.Wafer.Chip import Chip, Device
+
+
+class MoveStagesDeviceWindow(Toplevel):
+    """
+    Window to move stages to device
+    """
+
+    def __init__(
+        self,
+        master,
+        mover: Type[MoverNew],
+        chip: Type[Chip]
+    ) -> None:
+        """
+        Constructor for new move to device Window.
+
+        Parameters
+        ----------
+        master : Tk
+            Tk instance of the master toplevel
+        mover : Mover
+            Instance of the current mover.
+        mover : Chip
+            Instance of the current chip.
+
+        Raises
+        ------
+        RuntimeError
+            If mover cannot move absolutely
+            If chip is None.
+        """
+        self.mover: Type[MoverNew] = mover
+        self.chip: Type[Chip] = chip
+
+        if not self.mover.can_move_absolutely:
+            raise RuntimeError(
+                f"Cannot perform absolute movement, not all active stages are calibrated correctly. "
+                "Note for each stage a coordinate transformation must be defined.")
+
+        if self.chip is None:
+            raise RuntimeError(
+                "'No chip file imported before moving to device. Cannot move to device without chip file present.")
+
+        super(MoveStagesDeviceWindow, self).__init__(master)
+
+        # Set up window
+        self.title("Move Stages to Device")
+        self.geometry('{:d}x{:d}+{:d}+{:d}'.format(1000, 400, 200, 200))
+        self.protocol('WM_DELETE_WINDOW', self.destroy)
+
+        # Build window
+        self._main_frame = Frame(self, borderwidth=0, relief=FLAT)
+        self._main_frame.pack(side=TOP, fill=BOTH, expand=True, padx=10)
+
+        hint = "Automatically move the stages to a device selected below.\n"\
+               "The stages are lifted before and lowered again later."
```

Comment:
Suggestion for improvement of text:
```suggestion
        hint = "Automatically move the stages to a device selected below.\n"\
               f"The stages are z-lifted by {<var>:.0f}nm before the lateral movement and lowered again afterwards.\n"
```